components/Comment/CommentSidebarToggle.tsx

```diff
@@ -0,0 +1,79 @@
+import { css, StyleSheet } from "aphrodite";
+import colors from "./lib/colors";
+import { faComments } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import ResearchCoinIcon from "../Icons/ResearchCoinIcon";
+
+type Args = {
+  setIsOpen: Function;
+  isOpen: boolean;
+  bountyAmount: number;
+  commentCount: number;
+  isReady: boolean;
+};
+
+const CommentSidebarToggle = ({ isOpen, setIsOpen, isReady, bountyAmount = 0, commentCount = 0 }: Args) => {
+  const hasActiveBounties = bountyAmount > 0;
+  const bountyDisplayVal = (bountyAmount > 1000 ? bountyAmount / 1000 : bountyAmount).toFixed(0);
```

Comment:
This formatting is a bit different.

It will display:

"7K" instead of 7,000 on the toggle switch

![image](https://user-images.githubusercontent.com/802819/225765378-110ec170-d57a-4c35-8fec-ec17fbf2535f.png)

Reply:
instead of defining this ourselves, can we use the numeral library? that way it can be 7.2k, 7.5k, ...

http://numeraljs.com/

Reply:
Yeah that makes sense. I like it. I will do it in my followup PR